### ISSUE TITLE
clientStatus is null in get inbound in api

### ISSUE THREAD

**Issue by mostafa9112** [MHSanaei/3x-ui#41]
when i'm using this address for get an inbound , clientStatus is null
https://server.com/xui/API/inbounds/get/1
plz fix this

and plz add getClient to api

Your panel is the most complete compared to others, please add this, it will be great

**Comment by alipour66m** [comment]
اقای سنایی . به علی رضا دادید پنلو اپدیت کرد تقریبا میشه گفت زده کل ملتو بدبخت کرده ... ربات ها هه از کار افتادن ... چیکار کرده این بشررر؟

**Comment by alireza0** [comment]
Hello @mostafa9112 
"/API/inbounds/get/:id" returns only inbound information (stats are in separated table).
To get all the information together, you should use "/API/inbounds". Then you can divide all desired informaiton including clients and their stats.

**Comment by mostafa9112** [comment]
Yes, it is true, but when all inbounds are called, the clientstatus is also displayed based on each inbound
If I knew how to program Go, I would definitely do it because I am a php programmer and I'm sure it won't be difficult

As I said above, it increases the speed of working with the api

**Comment by alipour66m** [comment]
امکان داره تغییراتی که در این زمینه ایجاد کردید را مطابق نسخه اورجینال برگردانید؟

دچار مشکل شدیم همه
